crates/wasi-common/src/lib.rs

```diff
@@ -98,6 +98,9 @@ pub use sched::{Poll, WasiSched};
 pub use string_array::{StringArray, StringArrayError};
 pub use table::Table;
 
+pub mod blockless;
+pub use blockless::*;
```

Comment:
personally dont think we should pollute global scope as it becomes difficult to know at glance where functionality is coming from.

Reply:
I think you are right, I will do it later.